modules/master-node-sizing.adoc

```diff
@@ -6,34 +6,37 @@
 [id="master-node-sizing_{context}"]
 =  Control plane node sizing
 
-The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing. The control plane tests create the following objects across the cluster in each of the namespaces depending on the node counts:
+The control plane node resource requirements depend on the number and type of nodes and objects in the cluster. The following control plane node size recommendations are based on the results of a control plane density focused testing, or *Cluster-density*. This test creates the following objects across a given number of namespaces:
```

Comment:
number of nodes, type of nodes, and objects in the cluster. 

Instead of using ** around Cluster-density, use underscores to emphasize the first appearance of a new term: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference